### ISSUE TITLE
Expand arrow icon disappears from certain folder when I start Trilium

### ISSUE THREAD

**Issue by bwat47** [zadam/trilium#562]
Since the 0.32 update, I noticed that whenever I start trilium, the expand arrow icon disappears from one of my folders (general notes) shortly after trilium is opened:

![image](https://user-images.githubusercontent.com/1442520/59037738-df2b8800-883f-11e9-8a81-cb8fe45073d3.png)

I haven't been able to nail down any steps to re-create this with another folder. The only thing of note about this folder is that is has many things under it (probably more than any of the other folders)

If I reload the front end, the arrow re-appears. However, I noticed that it can disappear again if I click on any search results for notes contained within that folder. Again, for some reason it only seems to happen with this folder

**Comment by bwat47** [comment]
Actually, I found one way to re-create this with another folder (the calendar folder):

1. Create a code note for 'Today' (focuses current day in the calendar):

api.addButtonToToolbar({
    title: 'Today',
    icon: 'calendar',
    shortcut: 'alt+ctrl+o',
    action: async function() {
        const todayDateStr = api.formatDateISO(new Date());

        const todayNote = await api.runOnServer(async todayDateStr => {
            return await api.getDateNote(todayDateStr);
        }, [todayDateStr]);

        api.activateNote(todayNote.noteId);
    }
});

2. Hit the shortcut for today

At this point, the arrow next to calendar disappears until I reload frontend

**Comment by bwat47** [comment]
My attempts to find consistent steps to re-create this have been pretty fruitless. Most commonly it's when I'm clicking a search result for a note that's within a folder. 

Here's an example of what happens: https://youtu.be/bKa-_WJlz9I

**Comment by zadam** [comment]
Thank you for efforts to recreate this. I saw this problem once myself but could not figure out what's the problem. I noticed that the expand/collapse was still possible through left-right keys, it was just display issue.

I suspect this is a bug of the underlying library (fancytree) used to display the tree since this expander is fully automatic and trilium doesn't manually control it.

I will try to experiment a bit myself to get this reproducible ...
